### DISCUSSION TITLE
How to get as many as people to check my repo, I really want to earn some comment and discussion about my app

### DISCUSSION THREAD

**Discussion by popcari** [community/community#141883] [Programming Help]
I have a personal project about expense tracker (the app using localStorage to store data). At the first sight, I just want to create my very own app that doesn't have any ad because it"s so annoying. 
So I created my own expense tracker app. Now I really want to get advise and any comment about my project
This is my repo 's link: https://github.com/popcari/react-exenpse-tracker-v2
Could someone check around and if you feel convenience to use non-ad expense tracker app, pls hit me a star, it helps a lot

**Comment by Himani0002** [comment]
Hello

**Reply by Himani0002** [reply]
Hii
